### ISSUE TITLE
Bug in Russia in 180 long. 

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/dcw-gmt#40]
The segments of Russia that go trough 180 are split in two (that is why a blue line is drawn). I will joint the segments.

I think #39 could wait for this fixed. 

`gmt coast -R170/190/60/75 -ERU+gred+pblue -png Russia -JQ10 -W -Bf`

![Russia](https://user-images.githubusercontent.com/17908951/145726059-44eab06d-83ed-40ec-ad11-a1ae485bbeb7.png)

**Comment by Esteban82** [comment]
Just for curiosity, is there  way to do it with GMT? > > Something in gmt spatial maybe?

**Comment by PaulWessel** [comment]
No, I think that gap is explicitly closed.  If it were open then gmtconnect would do it, but here I think you just have to manually identify the two pieces, search for the 180 or -180 records and manually stitch.

**Comment by Esteban82** [comment]
I fix the polygon. Now there are 2 less segments (> 172 and 1027). Is it necessary to fix the number of the other segments? Is any way to do it easily?

BTW, I found similar errors in Fiyi and in Chukotka ("state" of Russia).

**Comment by Esteban82** [comment]
This is with the new version. 


![Russia](https://user-images.githubusercontent.com/17908951/145752299-cea270ed-1107-46b4-9cfd-000f183bed8c.png)
